src/main/java/uk/co/evoco/webdriver/configuration/WebDriverConfig.java

```diff
@@ -24,18 +24,19 @@
     private GridConfig gridConfig;
     private RunType runType;
     private List<String> exceptionsToHandleOnTolerantActions;
+    private String exceptionsWaitTimeOut;
     private Map<String, ObjectNode> browserPreferences;
 
+
+
     /**
-     *
      * @return BrowserType for the run from the configuration file
      */
     public BrowserType getBrowserType() {
         return browserType;
     }
 
     /**
-     *
```

Comment:
These need adding back in - javadocs convention is to have a blank line first which is where the description goes

Reply:
added the blank line